setup.cfg

```diff
@@ -0,0 +1,14 @@
+# All configuration for plugins and other utils is defined here.
+# Read more about `setup.cfg`:
+# https://docs.python.org/3/distutils/configfile.html
+
+[flake8]
```

Comment:
Thats right. I removed `flake8`.

I agree that the pylint configuration is quite large and makes the `setup.cfg` quite confusion. I suggest to keep `setup.cfg` because it is a standardized place for configuration where people can quickly take a look at what is configured without knowing which tool names to look for and additionally add a note in `setup.cfg` pointing to `.pylintrc`.

Reply:
Good idea with the note. I also added the mypy config (which is rather small) to `setup.cfg` and added that this only is the config for Python utils (should be clear, but as we have separate configs for ESLint etc. I think it is better not to state "All configuration for plugins and other utils is defined here.", but limit this statement to the Python code).